flink-libraries/flink-gelly-examples/build.gradle.kts

```diff
@@ -2,9 +2,12 @@ dependencies {
     implementation(project(":flink-java"))
     implementation(project(":flink-clients"))
     implementation(project(":flink-scala"))
+    implementation(project(":flink-runtime"))
```

Comment:
I think most projects don't have a `flink-runtime` dependency, or it should always be "provided". This would mean `compileOnly` here.